elser/README.md

```diff
@@ -0,0 +1,133 @@
+## ELSER Track
```

Comment:
I wonder if we should focus on ELSER only in this track. I'd prefer to name it msmarco-passage and add multiple challenges to report (ELSER, BM25, dense vector, hybrid search, ...)? We can start with ELSER in this PR but leave room for others in follow ups.